.github/workflows/push.yml

```diff
@@ -168,3 +168,26 @@ jobs:
           for file in ./bundle/internal/schema/testdata/fail/*.yml; do
             ajv test -s schema.json -d $file --invalid -c=./keywords.js
           done
+
+  validate-python-codegen:
+    needs: cleanups
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+      - name: Install uv
+        uses: astral-sh/setup-uv@0c5e2b8115b80b4c7c5ddf6ffdd634974642d182 # v5.4.1
+        with:
+          version: "0.6.5"
+
+      - name: Verify that python/codegen is up to date
+        working-directory: experimental/python
+        run: |
+          if ! ( make codegen && git diff --exit-code ); then
+            echo "Generated Python code is not up-to-date. Please run 'pushd experimental/python && make codegen' and commit the changes."
+
+            # TODO block PR if this fails once diffs are fixed
+            # exit 1
+          fi
```

Comment:
Are the tests run on push?

Reply:
There is a little bit of a problem, if codegen itself crashed CI job will pass now, let me fix it